### PR TITLE
Enable eval pipeline in new tenant

### PR DESCRIPTION
**Pull request by markus-hinsche**
**Status quo:** We changed Service connection / Service Principle to the new tenant. Now the pipeline will always run in the new tenant. 

**Changes:**
- `setuptools` and `wheel` are needed now otherwise the conda installation in the AzureML docker build fails
- run experiements again in new tenant (they get new run IDs)
- change experiment names and run ids so they get found in the new tenant
- Disable filter config because it would not yet work

Future work: we have to bring the standing/lying classification model in the new tenant so we can reference it in the pipeline again (@prajurock )

### CODE REVIEW

File: src/common/evaluation/QA/eval_depthmap_models/src/qa_config_filter.py [hunk 1]
```diff
@@ -3,9 +3,7 @@
 # Details of model used for evaluation
 MODEL_CONFIG = Bunch(dict(
     EXPERIMENT_NAME='q3-depthmap-plaincnn-height-95k',
-
     RUN_ID='q3-depthmap-plaincnn-height-95k_1597988908_42c4ef33',  # Run 3
-    # RUN_ID = 'q3-depthmap-plaincnn-height-95k_1600451633_cb44f6db',     #Run 17
 
     INPUT_LOCATION='outputs',
     NAME='best_model.h5',
```
Comment (prajurock) [inline src/common/evaluation/QA/eval_depthmap_models/src/qa_config_filter.py:9]:
```suggestion
    NAME='best_model.ckpt',
```

File: src/common/evaluation/QA/eval_depthmap_models/src/qa_config_height.py [hunk 2]
```diff
@@ -3,9 +3,7 @@
 # Details of model used for evaluation
 MODEL_CONFIG = Bunch(dict(
     EXPERIMENT_NAME='q3-depthmap-plaincnn-height-95k',
-
-    # RUN_ID='q3-depthmap-plaincnn-height-95k_1597988908_42c4ef33',  # Run 3
-    RUN_ID='q3-depthmap-plaincnn-height-95k_1600451633_cb44f6db',  # Run 17
+    RUN_ID='q3-depthmap-plaincnn-height-95k_1610709896_ef7f755d',  # Run 5
 
     INPUT_LOCATION='outputs',
     NAME='best_model.h5',
```
Comment (prajurock) [inline src/common/evaluation/QA/eval_depthmap_models/src/qa_config_height.py:9]:
```suggestion
    NAME='best_model.ckpt',
```

File: src/common/evaluation/QA/environment.yml [hunk 3]
```diff
@@ -2,6 +2,8 @@ name: CGM_QA_Pipeline
 channels:
   - conda-forge
 dependencies:
+  - setuptools=50.3.0
```
Comment (prajurock) [inline src/common/evaluation/QA/environment.yml:5]: Why you add this?

**Review by prajurock — CHANGES_REQUESTED** [review]

**Review by prajurock — APPROVED** [review]

File: src/common/evaluation/QA/environment.yml (hunk 3 above)
Comment (markus-hinsche) [reply src/common/evaluation/QA/environment.yml:5]: The PR description explains this.

File: src/common/evaluation/QA/test-pipeline.yml [hunk 4]
```diff
@@ -3,8 +3,6 @@ trigger:
     include:
     - releases/*
```
Comment (markus-hinsche) [inline src/common/evaluation/QA/test-pipeline.yml:5]:
```suggestion

pr: none

```

**Comment by markus-hinsche** [comment]
**height Dropout model**
old tenant(run17)
	min(val_mae): 2.21cm
new tenant(Run5):
	min(val_mae): 2.28cm

**Pull request merged**